src/backend/joanie/settings.py

```diff
@@ -548,10 +549,20 @@ class Test(Base):
     PASSWORD_HASHERS = [
         "django.contrib.auth.hashers.MD5PasswordHasher",
     ]
+    USE_SWAGGER = True
```

Comment:
Sorry, I don't get it…
`USE_SWAGGER` is defined in `Development` only

Reply:
Indeed I thought it was in the `Base` class line 494
But you must define it in the `Base`, otherwise in any other environment this settings will not exists and we will have an error in the `urls.py` when `settings.USE_SWAGGER` will be called.